cloud_service/installing_cloud_ocp/cloud-install-operator.adoc

```diff
@@ -0,0 +1,16 @@
+:_content-type: ASSEMBLY
+[id="cloud-install-operator"]
+= Install the Operator
+include::modules/common-attributes.adoc[]
+:context: cloud-install-operator
+
+
+[role="_abstract"]
+Install the {product-title-short} Operator on your secured clusters.
```

Comment:
Can we delete this one-liner because it looks like a repetition of the title? WDYT?